code/src/index.css

```diff
@@ -5,9 +5,85 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
+  color: white;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.Container{
+  border: solid black;
+  margin: 25px;
+  background-color: black; 
+  display: flex;
+  justify-content: center;
+}
+
+
+.img-container{
+  border: black;
+  position: relative;
+}
+
+
+p{
+   color:gainsboro;
+}
+
+.album-wrapper {
```

Comment:
it is super nice that you have managed to get the site responsive without any media queries, good job! However, when you view the site in mobile/tablet size right now the albums are not centered. This is solved with a justify-content: center right here! :)